src/app/(proper_react)/(redesign)/(public)/LandingView.tsx

```diff
@@ -43,226 +44,229 @@ export type Props = {
 
 export const View = (props: Props) => {
   return (
-    <main className={styles.wrapper}>
-      <nav className={styles.nav}>
-        <h1>
-          <Image
-            className={styles.logo}
-            src={MonitorLogo}
-            alt={props.l10n.getString("public-nav-name")}
-          />
-        </h1>
-        <SignInButton />
-      </nav>
-      <header className={styles.hero}>
-        <div className={styles.heroContent}>
-          <h1>{props.l10n.getString("landing-all-hero-title")}</h1>
-          <p>
-            {props.l10n.getString(
-              props.eligibleForPremium
-                ? "landing-premium-hero-lead"
-                : "landing-all-hero-lead",
-            )}
-          </p>
-          {props.eligibleForPremium && props.scanLimitReached ? (
-            <ScanLimit />
-          ) : (
-            <SignUpForm
-              scanLimitReached={props.scanLimitReached}
-              isHero
-              eligibleForPremium={props.eligibleForPremium}
-              signUpCallbackUrl={`${process.env.SERVER_URL}/user/dashboard/`}
-              eventId={{
-                cta: "clicked_get_scan_header",
-                field: "entered_email_address_header",
-              }}
+    <>
+      <AccountDeletionNotification />
+      <main className={styles.wrapper}>
```

Comment:
This looks like it's a huge change, but it only increases the indentation level to wrap it in `<></>`, and adds `<AccountDeletionNotification />` to the front.